### PR TITLE
Integrate template icons

### LINKED ISSUE #106: Integrate template icons in Cloud Modeler
__What should we do?__

Make the new template icons available in the Cloud Modeler
* [ ] bump `zeebe-bpmn-moddle`
* [ ] bump `bpmn-js-properties-panel`
* [ ] integrate `@bpmn-io/element-templates-icons-renderer`

__Why should we do it?__

Child of https://github.com/bpmn-io/internal-docs/issues/445

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #106 
Depends on https://github.com/bpmn-io/element-templates-icons-renderer/pull/2
Depends on https://github.com/bpmn-io/bpmn-js-properties-panel/pull/641

### CODE REVIEW

**Comment by barmac** [comment]
The way the icons are implemented means:
1. I get both icons on canvas and in the properties panel header when I use `camunda-bpmn-js`
2. Same as above if I install `bpmn-js-properties-panel` and `@bpmn-io/element-templates-icons-renderer` in a project without `camunda-bpmn-js`
3. My properties panel displays the icon in the header but I cannot see it on canvas when I build custom `bpmn-js` project without `@bpmn-io/element-templates-icons-renderer`

Do we intend to keep (3)? Perhaps the icons renderer should be required as a part of the element templates module -> to be extracted to a separate element templates package in the future.

**Comment by pinussilvestrus** [comment]
I thought about this one and given the current situation (element-templates as part of the properties panel) I found it strange to include rendering parts in the properties panel. However, I'd love to see the [element templates be a standalone module](https://github.com/bpmn-io/internal-docs/issues/503). We should definitely tackle this in Q2 IMHO.

**Comment by MaxTru** [comment]
@pinussilvestrus is out sick today, but this is super well prepared and the dependencies are met. I am marking this as ready for review.

**Comment by barmac** [comment]
We need to release and integrate another alpha of bpmn-js-properties-panel to merge it.

**Comment by MaxTru** [comment]
Back to draft. I realized, we should also bump prop-panel to integrate the [latest fix](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/652)

**Comment by MaxTru** [comment]
Icons arrived
Screenshot_20220405_112843

File: lib/camunda-cloud/Modeler.js [hunk 1]
```diff
@@ -60,7 +62,8 @@ Modeler.prototype._camundaCloudModules = [
   rulesModule,
   zeebePropertiesProviderModule,
   cloudElementTemplatesPropertiesProvider,
-  zeebeModdleExtension
+  zeebeModdleExtension,
+  iconRendererModule
```
Comment (nikku) [inline lib/camunda-cloud/Modeler.js:66]: To consider: Also add this to _Cloud Viewer_, _Cloud Navigated Viewer_, if we provide such things.

We want icons to be visible, end-to-end (cf. https://github.com/bpmn-io/internal-docs/issues/445).

Comment (MaxTru) [reply lib/camunda-cloud/Modeler.js:66]: Generally yes, but afaik camunda-bpmn-js does not provide Viewers. Viewer consumers so far just use bpmn-js vanilla.

Considering how we bundle the custom renderer, we might want to change this, and offer a camunda-bpmn-js viewer as well.

Comment (nikku) [reply lib/camunda-cloud/Modeler.js:66]: Let us leave it as is and create a follow up for this FR.

Comment (MaxTru) [reply lib/camunda-cloud/Modeler.js:66]: https://github.com/camunda/camunda-bpmn-js/issues/110

**Comment by MaxTru** [comment]
Back to draft, waiting until we have https://camunda.slack.com/archives/CKGH9LR40/p1649155653797739 fixed

**Comment by MaxTru** [comment]
Works with fix :-) 
Screenshot_20220405_203603

**Review by MaxTru — APPROVED** [review]

**Pull request merged**